README.md

```diff
@@ -1,33 +1,51 @@
 # engineering-education
-A community-generated pool of educational content which is useful for engineers of every level. Published content can be found at [https://www.section.io/engineering-education/](https://www.section.io/engineering-education/).
+The Engineering Education Program at Section was created as a medium to provide students a friendly and inviting place to contribute to a real-world projects to help students better build up their professional portfolios and profiles (upon entering the DEV career space). 
 
-[ENROLL HERE](https://docs.google.com/forms/d/e/1FAIpQLSfTbj3kqvEJEb5RLjqJurfbHa8ckzQx0CjRzaizblue9ZOK5A/viewform?usp=sf_link)
+Our goal is to allow our EngEd community the opportunity to increase and build upon the skills they have to build the features/enhancements the community wants to see in the EngEd program. At the same time have a place where they can build us a career profile and portfolio that can be leveraged when entering the workforce.
 
-### Program Purpose
-Section partners with university computer science programs across the globe to create a community-generated pool of content which is useful for engineers of every (any) level.
+We also offer students the opportunity to write content about related and approved topics (to build up there author profile and portfolio) about Information Technology Web-related services. (Could be something you are learning currently or a write-up on any EngEd Community Contribution they may have contributed). This will serve as a community-generated pool of educational content which is useful for engineers of every level. Published content can be found at [https://www.section.io/engineering-education/](https://www.section.io/engineering-education/).
```

Comment:
```suggestion
We also offer students the opportunity to write content about related and approved topics (to build up their author profile and portfolio) about Information Technology Web-related services. (Could be something you are learning currently or a write-up on any EngEd Community Contribution they may have contributed). This will serve as a community-generated pool of educational content which is useful for engineers of every level. Published content can be found at [https://www.section.io/engineering-education/](https://www.section.io/engineering-education/).
```